lib/benchee/formatters/console.ex

```diff
@@ -39,10 +38,20 @@ defmodule Benchee.Formatters.Console do
   ## Examples
 
   ```
-  iex> jobs = %{ "My Job" => %Benchee.Statistics{average: 200.0, ips: 5000.0,std_dev_ratio: 0.1, median: 190.0}, "Job 2" => %Benchee.Statistics{average: 400.0, ips: 2500.0, std_dev_ratio: 0.2, median: 390.0}}
-  iex> inputs = %{"My input" => jobs}
+  iex> scenarios = [
+  ...>   %Benchee.Benchmark.Scenario{
+  ...>     job_name: "My Job", input_name: "My input", run_time_statistics: %Benchee.Statistics{
+  ...>       average: 200.0, ips: 5000.0,std_dev_ratio: 0.1, median: 190.0
+  ...>     }
+  ...>   },
+  ...>   %Benchee.Benchmark.Scenario{
+  ...>     job_name: "Job 2", input_name: "My input", run_time_statistics: %Benchee.Statistics{
+  ...>       average: 400.0, ips: 2500.0, std_dev_ratio: 0.2, median: 390.0
+  ...>     }
+  ...>   }
+  ...> ]
```

Comment:
Thanks for reformatting :+1: 

Didn't know how to do this back when I wrote these :sweat_smile: